### PR TITLE
moving CI to github actions

### PR DESCRIPTION
**Pull request by agroszer**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,12 +1,222 @@
-# shoobx.mocks3
--e . [test]
+#
```
[the remark this answers is not in the document]
Comment (agroszer) [reply requirements.txt:1]: Freezing just some packages seemed halfbaked... rather all or nothing.
Looks like I can get moto 4.1.3 working in the next PR, so we could get rid of freezing.

[the remark this answers is not in the document]
Comment (jordanvance) [reply requirements.txt:1]: right, i'm just asking if there's a make endpoint for freezing everything, or if i need to remember to run `pip freeze > requirements.txt`

[the remark this answers is not in the document]
Comment (agroszer) [reply requirements.txt:1]: `make pip-compile` does that fit or we want a `make freeze` ?

[the remark this answers is not in the document]
Comment (jordanvance) [reply requirements.txt:1]: nope that works.

**Pull request merged**